test/integration/receive_records_test.go

```diff
@@ -65,8 +67,11 @@ func (a *receiveRecordsSuite) TestReceiveRecords_sendAndReceiveWithImporter() {
 			break
 		}
 		c++
+		if c == recordsCount { // lastrecord
```

Comment:
is it required? when expStream.Recv() returns err, this line is unreachable?

Reply:
no, we have last generated nextpulse record